src/components/Cluster/ClusterDetail/V5ClusterDetailTableNodePoolScaling.tsx

```diff
@@ -45,20 +73,25 @@ const V5ClusterDetailTableNodePoolScaling: React.FC<IV5ClusterDetailTableNodePoo
       >
         <NodePoolsColumnHeader>Current</NodePoolsColumnHeader>
       </OverlayTrigger>
-      <OverlayTrigger
-        overlay={
-          <Tooltip id='spot-tooltip'>
-            {Constants.SPOT_NODES_EXPLNANATION}
-          </Tooltip>
-        }
-        placement='top'
-      >
-        <NodePoolsColumnHeader>Spot</NodePoolsColumnHeader>
-      </OverlayTrigger>
+
+      {provider === Providers.AWS && (
+        <OverlayTrigger
+          overlay={
+            <Tooltip id='spot-tooltip'>
+              {Constants.SPOT_NODES_EXPLNANATION}
```

Comment:
```suggestion
              {Constants.SPOT_NODES_EXPLANATION}
```

Reply:
Needs to be changed here too https://github.com/giantswarm/happa/pull/1710/files#diff-6419c763868fb79ac119dd7025a34a7dR43